src/grading-settings/GradingSettings.jsx

```diff
@@ -42,7 +42,6 @@ const GradingSettings = ({ courseId }) => {
   } = useCourseSettings(courseId);
   const {
     mutate: updateGradingSettings,
-    isLoading: savePending,
```

Comment:
There is a simpler fix here.

You just need to change this line to

```
    isPending: savePending,
```
and it will work. The reason is the new version of React Query uses `isPending` instead of `isLoading`, so `isLoading` is actually undefined.

If you put `// @ts-check` on the first line, you'll see that right away:

<img width="496" height="157" alt="Image" src="https://github.com/user-attachments/assets/0eaca180-b30b-4c47-8807-bd735cad268f" />